packages/hwloc/hwloc.spec

```
@@ -0,0 +1,78 @@
+Name: %{_cross_os}hwloc
+Version: 2.12.2
+Release: 1%{?dist}
+Summary: Portable hardware locality library
+URL: http://www.open-mpi.org/projects/hwloc/
+License: BSD-2-Clause
+Source0: https://download.open-mpi.org/release/hwloc/v2.12/hwloc-%{version}.tar.bz2
+
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}systemd-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: hwloc development libraries and headers
+Requires: %{name}
```

You can check the `.pc` file but often you need to include other devel dependencies here as runtime requirements:
```suggestion
Requires: %{name}
Requires: %{_cross_os}systemd-devel
```